EcoSoapBank/Pickups/NewPickupViewController.swift

```diff
@@ -0,0 +1,263 @@
+//
+//  NewPickupViewController.swift
+//  EcoSoapBank
+//
+//  Created by Jon Bash on 2020-08-17.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+import Combine
+
+
+class NewPickupViewController: UIViewController {
+    typealias Snapshot = NSDiffableDataSourceSnapshot<Int, NewCartonViewModel>
```

Comment:
Oooh, oooh, oooh! Diffable data source... I have not messed around with this API yet.